src/components/coin-row/SendSavingsCoinRow.js

```diff
@@ -1,24 +1,28 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import { css } from 'styled-components/primitives';
-import { colors, padding } from '../../styles';
+import { StyleSheet } from 'react-native';
+import { colors } from '../../styles';
 import { ButtonPressAnimation } from '../animations';
 import { Text } from '../text';
 import CoinName from './CoinName';
 import CoinRow from './CoinRow';
 
-const containerStyles = css`
-  padding-bottom: 18;
-  padding-left: 15;
-  padding-top: 6;
-`;
-
 const selectedHeight = 78;
 
-const selectedStyles = css`
-  ${padding(15, 15, 19, 15)};
-  height: ${selectedHeight};
-`;
+const sx = StyleSheet.create({
+  container: {
+    paddingBottom: 18,
+    paddingLeft: 15,
+    paddingTop: 6,
+  },
+  containerSelected: {
+    height: selectedHeight,
+    paddingBottom: 19,
+    paddingLeft: 15,
```

Comment:
Do you have any pref. of using paddingHorizontal vs paddingLeft & paddingRight when the value is the same?

Reply:
i think i prefer `paddingHorizontal` , nice catch on this ^